feat(connectors): add doc how connector failure (#1898)
bonitasoft/bonita-doc: modules/runtime/pages/connectors-execution.adoc

image::images/connector_execution_timeout.png[Connector execution with timeout]

=== Connector failure handling

When a connector fails, it is put as failed and the exception can be retrieved
using the xref:api:bpm-api.adoc#connector-failure[dedicated api].

> Then the connector can be manually replayed using the xref:api:bpm-api.adoc#activity-replay[activityReplay api].

Reviewer: ```suggestion
Then the connector can be manually be replayed using the xref:api:bpm-api.adoc#activity-replay[activityReplay api].
```

[NOTE]
====

> On rare case, when an error occurs on the connector, and the engine is unable to mark it as failed

Reviewer: ```suggestion
On rare cases, when an error occurs on a connector and the engine is unable to mark it as failed
```

> (might be caused by the engine be unable to get a new sql connection) , the connector will automatically be retried.

Reviewer: ```suggestion
(this might be caused by the engine being unable to get a new sql connection), the connector will automatically be retried.
```
